src/components/Attachments/AttachmentCarousel/CarouselItem.js

```diff
@@ -31,6 +32,9 @@ const propTypes = {
 
         /** Whether the attachment has been flagged */
         hasBeenFlagged: PropTypes.bool,
+
+        /** The id of the transaction related to the attachment */
+        transactionID: PropTypes.string,
```

Comment:
Where is this being passed in? (Trying to understand patterns for ECard transactions eReceipts). 

I'm having trouble finding where the URL is coming from, do you save it to the receipt key in the transaction object in the BE?

Reply:
Thank you! This was super helpful